packages/tooling/fast-tooling-react/src/form/custom-controls/css/color-picker-component/color-picker.template.ts

```diff
@@ -0,0 +1,116 @@
+import { html, ref } from "@microsoft/fast-element";
+import { ColorPicker } from "./color-picker";
+
+/**
+ * The template for the color picker component.
+ * @public
+ */
+export const ColorPickerTemplate = html<ColorPicker>`
+    <template
+        class="
+            ${x => (x.readOnly ? "readonly" : "")}
+        "
```

Comment:
If we're binding styles to this they could get overridden by React. We should likely update to target the attribute for readonly that's set on the component.

Reply:
I think this was copied over from something else it isn't actually needed. Removed.